Dockerfile

```diff
@@ -52,7 +45,7 @@ LABEL com.redhat.component="osp-director-operator-container" \
 
 ENV USER_UID=1001 \
     OPERATOR_TEMPLATES=/usr/share/osp-director-operator/templates/ \
-    WATCH_NAMESPACE=openstack,openshift-machine-api,openshift-sriov-network-operator
```

Comment:
is this intentional change for this effort? This change should likely be driven based on the operator change not a rework for layout/structure of the Dockerfile

Reply:
@yazug This is what I found in the midstream repo, but I can revert back to the upstream version of it.

Reply:
use upstream version please.